multinode-demo/client.sh

```diff
@@ -9,7 +9,9 @@ LEADER=$1
 COUNT=${2:-1}
 
 set -x
-rsync -v -e ssh "$LEADER"/{leader.json,mint-demo.json} .
+if [[ "$LEADER" != "localhost" ]]; then
+  rsync -v -e ssh "$LEADER"/{leader.json,mint-demo.json} .
```

Comment:
turns out that using rsync locally works fine too...  I've been doing:

mkdir client
cd client
../multinode-demo/client.sh .. 1

works fine

Reply:
Ah, nice. `./multinode-demo/client.sh . 1` works too (I wanted to run everything from the same directory), will update this